bin/sign

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+/*
+ * This Source Code is subject to the terms of the Mozilla Public License
+ * version 2.0 (the 'License'). You can obtain a copy of the License at
+ * http://mozilla.org/MPL/2.0/.
+ */
+/*
+ * This script is for signing the Min Vid add-on. If the current
+ * package version has already been signed, we download the signed xpi
+ * from amo, and move the files into the dist/ directory.
+ * If the package version has not been signed, sign the xpi, and move
+ * them into the /dist directory.
+ *
+ */
+var fs = require('fs');
+var path = require('path');
+var request = require('request');
+var jwt = require('jsonwebtoken');
+var manifestFilename = __dirname + '/../package.json';
+var manifest = require(manifestFilename);
```

Comment:
you can probably just do `var manifest = require('../package.json');` here instead.